delfin/drivers/fujitsu/eternus/eternus_stor.py

```diff
@@ -352,3 +391,548 @@ def parse_alert(context, alert):
+                    key_length = key_length + length_key + DIGITAL_CONSTANT \
+                        .ONE_INT
+                host_list.append(volume_list)
+        return host_list
+
+    def get_iscsi_host(self):
+        iscsi_list = []
+        iscsi_ids_str = self.cli_handler.exec_command(
+            consts.GET_HOST_ISCSI_NAMES)
+        block = True
+        if iscsi_ids_str:
+            iscsi_ids_arr = iscsi_ids_str.strip().replace('\r', '').split('\n')
+            for iscsi_ids_row_str in (iscsi_ids_arr or []):
+                if not iscsi_ids_row_str or \
+                        consts.CLI_STR in iscsi_ids_row_str:
+                    continue
+                if consts.HOST_PATH_STATUS_SPECIFIC_ONE in iscsi_ids_row_str:
+                    block = False
+                    continue
+                if block:
+                    continue
+                iscsi_ids_row_arr = iscsi_ids_row_str.strip().split()
+                if len(iscsi_ids_row_arr) < consts.HOST_ISCSI_NAMES_SEVEN:
+                    continue
+                details = self.get_iscsi_details(
+                    iscsi_ids_row_arr[consts.HOST_ISCSI_NAMES_ZERO])
+                iscsi_d = {
+                    'iscsi_id': details.get('Host No.'),
+                    'name': details.get('Host Name'),
+                    'iqn': details.get('iSCSI Name'),
+                    'alias': details.get('Alias Name'),
+                    'address': None if
+                    consts.HOST_ISCSI_SPECIFIC_ONE in
+                    details.get('IP Address') else details.get('IP Address'),
+                    'os': details.get('Host Response Name')
+                }
+                iscsi_list.append(iscsi_d)
+        return iscsi_list
+
+    def get_iscsi_details(self, number):
+        details = {}
+        iscsi_details_str = self.cli_handler.exec_command(
+            consts.GET_HOST_ISCSI_NAMES_NUMBER.format(number))
+        if iscsi_details_str:
+            iscsi_ids_arr = iscsi_details_str.strip().replace('\r', '') \
+                .split('\n')
+            for row_str in (iscsi_ids_arr or []):
+                if not row_str or consts.CLI_STR in row_str:
+                    continue
+                iscsi_details_row_arr = row_str.strip().split('   ')
+                if len(iscsi_details_row_arr) < consts.HOST_ISCSI_NAMES_TWO:
+                    continue
+                key = row_str[:consts.HOST_ISCSI_DETAIL_EIGHTEEN].strip()
+                value = row_str[consts.HOST_ISCSI_DETAIL_EIGHTEEN:].strip()
+                details[key] = value
+        return details
+
+    def get_host_status(self):
+        status_d = {}
+        status_list = self.get_data(consts.GET_HOST_PATH_STATUS,
+                                    consts.HOST_PATH_STATUS_TOTAL)
+        for status_row in status_list:
+            if len(status_row) < consts.HOST_PATH_STATUS_TOTAL:
+                continue
+            host_name = status_row[consts.HOST_PATH_STATUS_NAME]
```

Comment:
The two variables are used only once, and the constant consts.HOST_PATH_STATUS_NAME can be read. Therefore, you do not need to define the two variables separately. You need to delete the two variables.
![image](https://user-images.githubusercontent.com/59107077/153790666-7da56550-3dc0-405b-b178-b419f372c3ad.png)

Reply:
I only used it once, but you can't use Arabic numerals directly in a function because it's a very common behavior, so you can think about it a little bit